druid/src/widget/textbox.rs

```diff
@@ -134,7 +135,7 @@ impl TextBox {
             EditAction::ModifySelection(movement) => self.move_selection(movement, text, true),
             EditAction::SelectAll => self.selection.all(text),
             EditAction::Click(action) => {
-                if action.mods.shift {
+                if action.mods.contains(Modifiers::SHIFT) {
```

Comment:
I don't love the ergonomics here, would be nice if we could just do `action.has_shift()` or something, or at least `action.modifiers.includes_shift()`.

This is maybe related to the idea of still having our own `KeyboardEvent` type.

Reply:
I agree with you that the bitflags ergonomics are a downgrade, but one reason I went with it is that it's very much standard Rust practice. I'm also worried about too much timtowtdi, especially if we start having methods on both `Modifiers` and types that contain it.